TTS/server/server.py

```diff
@@ -211,19 +213,23 @@ def mary_tts_api_voices():
         "{{ name }} {{ locale }} {{ gender }}\n", name=model_details[3], locale=model_details[1], gender="u"
```

Comment:
Here it would be helpful now to list all the available speaker names for multi-speaker models.